buildSrc/src/main/java/com/duckduckgo/gradle/IntermoduleDependencyManager.kt

```diff
@@ -39,6 +73,12 @@ class IntermoduleDependencyManager {
             println("Wiring up module: ${implModule.name}")
             val gradleModifier = BuildGradleModifier(File(implModule, BUILD_GRADLE))
 
+            // it was creating before
+            if (!gradleModifier.lineExists(PLACEHOLDER_API_DEPENDENCY)) {
+                println("[skip] ${implModule.name} already existed")
+                return
+            }
```

Comment:
don't think this is needed... isn't that what the next line `gradleModifier.removeDependency(PLACEHOLDER_API_DEPENDENCY)` handles anyway?